Command/RunParatestCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Liip\FunctionalTestBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+/**
+ * Command used to update the project.
+ */
+class RunParatestCommand extends ContainerAwareCommand
+{
+    private $container;
+    private $configuration;
+    private $output;
+    private $process = 5;
+    private $testDbPath;
+    private $phpunit = './bin/phpunit';
+
+    /**
+     * Configuration of the command.
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('test:run')
+            ->setDescription('Run phpunit tests with multiple process')
+        ;
+    }
+
+    protected function prepare()
+    {
+        $this->configuration = $this->getContainer()->hasParameter('liip_functional_test');
+        $paratestCfg = (!isset($this->configuration['paratest'])) ? array('process' => $this->process, 'phpunit' => $this->phpunit) : $this->configuration['paratest'];
+
+        $this->process = (!empty($this->configuration['process'])) ? $paratestCfg['process'] : $this->process;
+        $this->phpunit = (!empty($this->configuration['phpunit'])) ? $paratestCfg['phpunit'] : $this->phpunit;
+        $this->testDbPath = $this->getContainer()->get('kernel')->getRootDir();
+        $this->output->writeln("Cleaning old dbs in $this->testDbPath ...");
+        $createDirProcess = new Process('mkdir -p '.$this->testDbPath.'/cache/test/');
+        $createDirProcess->run();
+        $cleanProcess = new Process("rm -fr $this->testDbPath/cache/test/dbTest.db $this->testDbPath/cache/test/dbTest*.db*");
+        $cleanProcess->run();
+        $this->output->writeln("Creating Schema in $this->testDbPath ...");
+        $createProcess = new Process('php app/console doctrine:schema:create --env=test');
+        $createProcess->run();
+
+        $this->output->writeln('Initial schema created');
+        $populateProcess = new Process("php app/console doctrine:fixtures:load -n --fixtures $this->testDbPath/../src/overlord/AppBundle/Tests/DataFixtures/ORM/ --env=test");
+        $populateProcess->run();
+
+        $this->output->writeln('Initial schema populated, duplicating....');
+        for ($a = 0; $a < $this->process; ++$a) {
```

Comment:
What is the role of this code? What do you copy the SQLite databases? It seems to be inconsistent with the name returned in the [`ConnectionFactory`](https://github.com/liip/LiipFunctionalTestBundle/blob/5305d8e211e47e3c0ab3d8781fbe0b7b8e141ed0/Factory/ConnectionFactory.php#L27L40). Is `TEST_TOKEN` always a number which correspond to this loop?